distributed/worker.py

```diff
@@ -1069,6 +1073,18 @@ async def gather(self, comm=None, who_has=None):
             self.update_data(data=result, report=False)
             return {"status": "OK"}
 
+    def get_monitor_range(self, comm=None, start=0):
+        return self.monitor.range_query(start=start)
+
+    def get_monitor_recent(self, comm=None):
+        return self.monitor.recent()
+
+    def get_monitor_count(self, comm=None):
+        return self.monitor.count
+
+    def get_monitor_last_time(self, comm=None):
+        return self.monitor.last_time
```

Comment:
Sure! In that case, I think the resulting dictionary the scheduler returns should be of the format:

```python
{
    "<worker_address>": {
        "range_query": {...},
        "count": 23,
        "last_time": 16.54
    },
    ...
}
```

So that we don't have to bother with reformatting the results of the `gather()`.

Reply:
Didn't see your comment below - I'll opt for `scheduler.broadcast()` with a consolidated worker method, which will guarantee this formatting.